api/v1/utils/time.js

```diff
@@ -0,0 +1,7 @@
+/* jshint esversion: 6 */
+
+const MILLISECONDS_PER_SECOND = 1000;
```

Comment:
[Maybe we should move these math constants to one place.](https://github.com/HackIllinois/api-2017/blob/staging/api/config.js#L85)